EventFilter/HGCalRawToDigi/interface/Unpackertools.h

```diff
@@ -0,0 +1,16 @@
+#ifndef EventFilter_HGCalRawToDigi_interface_Unpackertools_h
```

Comment:
The CMS style (http://cms-sw.github.io/cms_coding_rules.html) suggests that subsequent words in a class/variable name should be capitalized, i.e. `UnpackerTools`. (This change should be propagated to the file names as well as the include guard here.)